### PR TITLE
Add 112's Summer Tournament 2019 article

### PR DESCRIPTION
**Pull request by zeusminus**
- Add `112's Tournaments` section in Index
- Add `112's Summer Tournament 2019` wiki article

### CODE REVIEW

**Comment by zeusminus** [comment]
Ready for review

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 1]
```diff
@@ -0,0 +1,656 @@
...
+- 112ST 2019
+---
+
+# 112's Summer Tournament 2019
+
+![112ST Logo](img/Logo.png)
+
+The **112's Summer Tournament 2019** (***112ST***) was a double-elimination 2v2 osu! tournament started with 1v1 qualifiers hosted by ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779) and ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852). The tournament was only allowed for Turkish-speaking users regardless of their country flags. This was the fourth installment of 112's Tournaments series.
+
+## Tournament Schedule
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:14]:
```suggestion
## Tournament schedule
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 2]
```diff
@@ -0,0 +1,656 @@
...
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2019-06-05/2019-06-30 |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:18]:
```suggestion
| Registration phase | 2019-06-05/2019-06-30 |
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 3]
```diff
@@ -0,0 +1,656 @@
...
+| Qualifiers Stage | 2019-07-13/2019-07-14 |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:19]:
```suggestion
| Qualifiers | 2019-07-13/2019-07-14 |
```
Not necessary

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 4]
```diff
@@ -0,0 +1,656 @@
...
+| Finals | 2019-08-11/2019-08-16 |
+| Grand Finals | 2019-08-19/2019-08-22 |
+
+## Prizes
+
+| Placing | Prize(s) |
+| :-: | :-- |
+| ![Gold Crown](/wiki/shared/crown-gold.png "1st place") | Unique profile badge, 1 month of osu!supporter tag |
+
+![112's Summer Tournament 2019 Winning Team](img/Badge.png)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:32]:
```suggestion
![112's Summer Tournament 2019 Winner Badge](img/Badge.png)
```
More appropriate

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 5]
```diff
@@ -0,0 +1,656 @@
...
+
+## Organisation
+
+The 112's Summer Tournament 2019 was run by various community members by distributing the multitude of tasks into various fields of responsibility.
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:36]:
```suggestion
The 112's Summer Tournament 2019 was run by various community members.
```
Shorter

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 6]
```diff
@@ -0,0 +1,656 @@
...
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:40]:
```suggestion
| Manager | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
```
If you want to change this to Organiser, it's ok too.

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 7]
```diff
@@ -0,0 +1,656 @@
...
+| Map Selector | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_TR] [sentez](https://osu.ppy.sh/users/8816305) |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:41]:
```suggestion
| Map selector | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_TR] [sentez](https://osu.ppy.sh/users/8816305) |
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 8]
```diff
@@ -0,0 +1,656 @@
...
+| Streamer | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_NL] [goeo_](https://osu.ppy.sh/users/6832107), ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849) |
+| Commentator | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [bigbirb-](https://osu.ppy.sh/users/11285985), ![][flag_TR] [Ekin](https://osu.ppy.sh/users/2947405), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TR] [Lxran](https://osu.ppy.sh/users/3651081), ![][flag_TR] [mezelyus](https://osu.ppy.sh/users/5938859), ![][flag_NL] [goeo_](https://osu.ppy.sh/users/6832107) |
+| Designer | ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Statistician | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779) |
+| Wiki Editor | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:47]:
```suggestion
| Wiki editor | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) |
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 9]
```diff
@@ -0,0 +1,656 @@
...
+
+| Position | Member(s) |
+| :-- | :-- |
+| Manager | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Map Selector | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TH] [Seaweed](https://osu.ppy.sh/users/5151214), ![][flag_TR] [sentez](https://osu.ppy.sh/users/8816305) |
+| Referee | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [Entry](https://osu.ppy.sh/users/10213311), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_NL] [goeo_](https://osu.ppy.sh/users/6832107), ![][flag_TR] [LeBoum](https://osu.ppy.sh/users/8700026), ![][flag_TR] [oralekin](https://osu.ppy.sh/users/7631823), ![][flag_TR] [ozge1907](https://osu.ppy.sh/users/6511453), ![][flag_PL] [P a t r i c k](https://osu.ppy.sh/users/6814521) |
+| Streamer | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_NL] [goeo_](https://osu.ppy.sh/users/6832107), ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849) |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:43]:
```suggestion
| Streamer | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_NL] [goeo\_](https://osu.ppy.sh/users/6832107), ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849) |
```
Escaping

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 10]
```diff
@@ -0,0 +1,656 @@
...
+| Commentator | ![][flag_TR] [112servis](https://osu.ppy.sh/users/3953470), ![][flag_TR] [bigbirb-](https://osu.ppy.sh/users/11285985), ![][flag_TR] [Ekin](https://osu.ppy.sh/users/2947405), ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779), ![][flag_TR] [Lxran](https://osu.ppy.sh/users/3651081), ![][flag_TR] [mezelyus](https://osu.ppy.sh/users/5938859), ![][flag_NL] [goeo_](https://osu.ppy.sh/users/6832107) |
+| Designer | ![][flag_TR] [Sibyl](https://osu.ppy.sh/users/10440852) |
+| Statistician | ![][flag_TR] [heyronii](https://osu.ppy.sh/users/5642779) |
+| Wiki Editor | ![][flag_TR] [Zeus-](https://osu.ppy.sh/users/5464437) |
+
+## Links
+
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/917454)
+- [112's Tournaments Discord Server](https://discord.gg/g3nqqeZ)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:52]:
```suggestion
- [112's Tournaments Discord server](https://discord.gg/g3nqqeZ)
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 11]
```diff
@@ -0,0 +1,656 @@
...
+- [Livestream](https://www.twitch.tv/osu_turkiye)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1d6JqZWqnZ32nFCgGMcXz3Pc4KyE1iLmU5KpTIg6TEhE/edit)**
+
+---
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:56]:
```suggestion

```
Not necessary, delete all line breaks

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 12]
```diff
@@ -0,0 +1,656 @@
...
+| #8 | ![][flag_TR] [Iron Man](https://osu.ppy.sh/users/4242381) | 9,2 |
+| #9 | ![][flag_TR] [dragonsaga](https://osu.ppy.sh/users/4982690) | 9,6 |
+| #10 | ![][flag_TR] [exporage](https://osu.ppy.sh/users/5573525) | 10,8 |
+| #11 | ![][flag_TR] [Ruushin](https://osu.ppy.sh/users/4696304) | 11 |
+| #12 | ![][flag_TR] [Ekin](https://osu.ppy.sh/users/2947405) | 11 |
+| #13 | ![][flag_TR] [Cynesta](https://osu.ppy.sh/users/8674027) | 13 |
+| #14 | ![][flag_TR] [huckmen444](https://osu.ppy.sh/users/5019334) | 13,2 |
+| #15 | ![][flag_TR] [Mr\_Ege](https://osu.ppy.sh/users/9542170) | 13,8 |
+| #16 | ![][flag_TR] [Valiente](https://osu.ppy.sh/users/2617985) | 15,2 |
+| E | ![][flag_TR] [Aga Kalk Sahur](https://osu.ppy.sh/users/8212609) | 15,2 |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:80]:
```suggestion
| E | ![][flag_TR] [Aga Kalk Sahur](https://osu.ppy.sh/users/8212609) | 15,2 |
```
May want to add a little footnote.
Something like:
*E: Eliminated

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 13]
```diff
@@ -0,0 +1,656 @@
...
+
+---
+
+## Participants
+
+### 1-15K Seed
+
+| Seed Ranking | Players | Qualifier Points |
+| :-- | :-- | :-- |
+| #1 | ![][flag_DE] [-Kanzaki](https://osu.ppy.sh/users/1652478) | 1,6 |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:64]:
```suggestion
| #1 | ![][flag_DE] [-Kanzaki](https://osu.ppy.sh/users/1652478) | 1.6 |
```
Use dots not comma (international)

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 14]
```diff
@@ -0,0 +1,656 @@
...
+- [Livestream](https://www.twitch.tv/osu_turkiye)
+- **[Statistics sheet](https://docs.google.com/spreadsheets/d/1d6JqZWqnZ32nFCgGMcXz3Pc4KyE1iLmU5KpTIg6TEhE/edit)**
+
+---
+
+## Participants
+
+### 1-15K Seed
+
+| Seed Ranking | Players | Qualifier Points |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:62]:
```suggestion
| Seed ranking | Players | Qualifier points |
```
Sentence case.
Apply all changes within this topic.

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 15]
```diff
@@ -0,0 +1,656 @@
...
+| #13 | ![][flag_TR] [HayunGuys](https://osu.ppy.sh/users/9693852) | 13,4 |
+| #14 | ![][flag_TR] [amdinstartor](https://osu.ppy.sh/users/10505284) | 13,4 |
+| #15 | ![][flag_TR] [karyuunatsu](https://osu.ppy.sh/users/12575190) | 14 |
+| #16 | ![][flag_TR] [Horoz](https://osu.ppy.sh/users/11614447) | 14 |
+| E | ![][flag_TR] [L Lones](https://osu.ppy.sh/users/4954493) | 14,6 |
+| E | ![][flag_TR] [Mikazu](https://osu.ppy.sh/users/12474487) | 17 |
+
+Only first 16 players from each seed are qualified, the rest is eliminated.
+
+---
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:111]:
```suggestion

```
Delete LB

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 16]
```diff
@@ -0,0 +1,656 @@
...
+
+## Teams
+
+After Group matches, those who get through to Quarterfinals are obliged to form a team with a player from the other seed.
+
+| Team Name | 1 - 15K Player | 15K+ Player |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:117]:
```suggestion
| Team | 1 - 15K seed | 15K+ seed |
```

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 17]
```diff
@@ -0,0 +1,656 @@
...
+| Kwk Yelleri | ![][flag_TR] [KaraElmas](https://osu.ppy.sh/users/9300293) | ![][flag_TR] [mezelyus](https://osu.ppy.sh/users/5938859) | 
+| KFC Managers | ![][flag_TR] [dragonsaga](https://osu.ppy.sh/users/4982690) | ![][flag_TR] [pyth0nix](https://osu.ppy.sh/users/10922849) |
+| eTi dasmjasdk | ![][flag_TR] [edizberkserbest](https://osu.ppy.sh/users/9256771) | ![][flag_TR] [tingirmin](https://osu.ppy.sh/users/9027514) |
+| egemen ve aimless | ![][flag_TR] [egemenbsrms](https://osu.ppy.sh/users/4520477) | ![][flag_TR] [aimlessplayr](https://osu.ppy.sh/users/4903088) |
+| İsim bulamadık pingleme artık | ![][flag_TR] [AkashiDagara](https://osu.ppy.sh/users/6118485) | ![][flag_TR] [empirasu](https://osu.ppy.sh/users/12500084) |
+| Yürü baba fingo | ![][flag_TR] [Iron Man](https://osu.ppy.sh/users/4242381) | ![][flag_TR] [emptypudding](https://osu.ppy.sh/users/11003085) |
+
+---
+
+![112ST 2019 Podium](img/Podium.png)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:130]: Delete the line breaks. Replace with

```
## Podium

the image
```

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 18]
```diff
@@ -0,0 +1,656 @@
...
+
+---
+
+## Mappools
+
+### Grand Finals
+
+**[Download the map pack here!](https://112.s-ul.eu/yUBjoWpe)**
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:138]:
```suggestion
**[Download the map pack here!](https://112.s-ul.eu/yUBjoWpe)**
```
You can also hint the expected file size
e.g. [Download the map pack here! (60 MB)]

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 19]
```diff
@@ -0,0 +1,656 @@
...
+- Tiebreaker
+  - **[gmtn. vs. kozato (fw. LUZE) - squartatrice (Morinaga) \[Skystar’s Descent\]](https://osu.ppy.sh/beatmapsets/679272#osu/1436329)**
+
+### Finals
+
+**[Download the map pack here!](https://112.s-ul.eu/MBhzkPlU)**
+
+- NoMod
+  - [nano - Nevereverland (zhu) \[Starset’s Redemption\]](https://osu.ppy.sh/beatmapsets/621438#osu/1507079)
+  - [sky_delta vs Street – Load of the Ciel (Rokkea) \[BMD’s Extra\]](https://osu.ppy.sh/beatmapsets/340527#osu/861191)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:171]:
```suggestion
  - [sky\_delta vs Street – Load of the Ciel (Rokkea) \[BMD’s Extra\]](https://osu.ppy.sh/beatmapsets/340527#osu/861191)
```
Escaping

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 20]
```diff
@@ -0,0 +1,656 @@
...
+  - **[NOMA – Brain Power Long Version (Skystar) \[Overdrive\]](https://osu.ppy.sh/beatmapsets/432822#osu/933228)**
+
+### Quarterfinals
+
+**[Download the map pack here!](https://112.s-ul.eu/yIQMxrAl)**
+
+- NoMod
+  - [Son Feci Bisiklet – Viva La Vadi (Pati) \[Green Valley\]](https://osu.ppy.sh/beatmapsets/460386#osu/986011)
+  - [The Eden Project - Lost (Bearizm) \[Escapism\]](https://osu.ppy.sh/beatmapsets/247747#osu/835941)
+  - [sky_delta - Kreuz (Side) \[Another\]](https://osu.ppy.sh/beatmapsets/347551#osu/785113)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:230]:
```suggestion
  - [sky\_delta - Kreuz (Side) \[Another\]](https://osu.ppy.sh/beatmapsets/347551#osu/785113)
```
Escaping

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 21]
```diff
@@ -0,0 +1,656 @@
...
+- HardRock
+  - [Fujiwara Chika (CV: Kohara Konomi) – Chikatto Chika Chika (Andrea) \[Sekai’s IQ <3\]]( https://osu.ppy.sh/beatmapsets/962521#osu/2015394)
+  - [YURiKA – Kyoumen no Nami (Vell) \[Expert\]](https://osu.ppy.sh/beatmapsets/731108#osu/1542973)
+  - [sakuzyo - Altale (toybot) \[Bonzi’s Extra\]](https://osu.ppy.sh/beatmapsets/228815#osu/1012279)
+- DoubleTime
+  - [Mili - Rightfully (Okoratu) \[Hard\]](https://osu.ppy.sh/beatmapsets/875117#osu/1874011)
+  - [96neko - ELECT (Lasse) \[Insane\]](https://osu.ppy.sh/beatmapsets/716122#osu/1517345)
+  - [3L – Spring of Dreams (impossiblexu) \[Karen\]](https://osu.ppy.sh/beatmapsets/27875#osu/97338)
+- FreeMod
+  - [A.SAKA – Nanatsu Issenzakura (yf_bmp) \[Gust’s Insane\]](https://osu.ppy.sh/beatmapsets/513731#osu/1350672)
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:246]:
```suggestion
  - [A.SAKA – Nanatsu Issenzakura (yf\_bmp) \[Gust’s Insane\]](https://osu.ppy.sh/beatmapsets/513731#osu/1350672)
```
Escaping

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 22]
```diff
@@ -0,0 +1,656 @@
...
+  - [Memme – Geurida (NeilPerry) \[Mellifluous Miracle\]](https://osu.ppy.sh/beatmapsets/688183#osu/1456417)
+  - [Major Lazer – Sua Cara (feat. Anitta & Pabllo Vittar) (handsome) \[byfar’s extra\]](https://osu.ppy.sh/beatmapsets/720615#osu/1665735)
+- Hidden
+  - [ak+q – Axium Crisis (MrSergio) \[Collapse\]](https://osu.ppy.sh/beatmapsets/812551#osu/1704329)
+- HardRock
+  - [Nekomata Master+ – encounter (Sing) \[Frey’s Expert\]](https://zeu.s-ul.eu/Lzk6C8ux.osz)
+- DoubleTime
+  - [Hate vs Brilliance – Qubism (Regraz) \[Elexus’ HYPER\]](https://osu.ppy.sh/beatmapsets/524503#osu/1802266)
+
+---
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:332]:
```suggestion

```
Delete LB

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 23]
```diff
@@ -0,0 +1,656 @@
...
+
+## Match results
+
+### Grand Finals
+
+Saturday, 17 August 2019:
+
+| Team 1 | | | Team 2 | Match link |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:340]:
```suggestion
| Team 1 |  |  | Team 2 | Match link |
```
Use 2 cell padding. Apply for all.

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 24]
```diff
@@ -0,0 +1,656 @@
...
+| --: | :-: | :-: | :-- | :-- |
+| Kwk Yelleri | FF | **1** | **egemen ve aimless** | *-win by default-* |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:342]:
```suggestion
| Kwk Yelleri | FF | **1** | **egemen ve aimless** | *win by default* |
```
You can let the dash go.

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 25]
```diff
@@ -0,0 +1,656 @@
...
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **KaFein** | **5** | 1 | İsim bulamadık pingleme artık | [#1]( https://osu.ppy.sh/community/matches/53645819) |
+
+Sunday, 28 July 2019:
+
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| **bum patladın** | **1** | 0 | eTi dasmjasdk | FF |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:403]:
```suggestion
| **bum patladın** | **1** | 0 | eTi dasmjasdk | FF |
```
Use `*forfeit*`
Or `*win by default*`

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 26]
```diff
@@ -0,0 +1,656 @@
...
+| Team 1 | | | Team 2 | Match link |
+| --: | :-: | :-: | :-- | :-- |
+| İsim bulamadık pingleme artık | 3 | **5** | **KFC Managers** | [#1]( https://osu.ppy.sh/community/matches/53835518) |
+
+### Group Matches
+
+Saturday, 20 July 2019:
+
+| Group | Player 1 | | | Player 2 |
+| :--: | --: | :-: | :-: | :-- |
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:419]:
```suggestion
| :-: | --: | :-: | :-: | :-- |
```
I found three that has something like this. replace with only one dash for middle alignment

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 27]
```diff
@@ -0,0 +1,656 @@
...
+| C | **Mr\_Ege** | **4** | 0 | exporage | 
+| G | **empirasu** | **4** | 0 | HayunGuys | 
+
+---
+
+## Ruleset
+
+### Tournament Rules
+
+1. 112’s Summer Tournament 2019 is an osu!standard tournament only allowed for Turkish-speaking players regardless of their profile flags.
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:486]: Remove all the spacings in the ruleset

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 28]
```diff
@@ -0,0 +1,656 @@
...
+| :--: | --: | :-: | :-: | :-- |
+| B | **-Kanzaki** | **4** | FF | Cynesta | 
+| C | **Mr\_Ege** | **4** | 0 | exporage | 
+| G | **empirasu** | **4** | 0 | HayunGuys | 
+
+---
+
+## Ruleset
+
+### Tournament Rules
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:484]:
```suggestion
### Tournament rules
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 29]
```diff
@@ -0,0 +1,656 @@
...
+
+2. Participants **must** join the [Discord server](https://discord.gg/g3nqqeZ) for the duration of the tournament.
+
+3. Excluding commentators and streamers, tournament staff cannot participate in the tournament.
+
+4. Upon the conclusion of registration period, the list of participants will be submitted to the osu! Support Team for screening. Participants that fail the screening procedure will not be able to participate in the tournament. Please refer to [here](https://osu.ppy.sh/help/wiki/osu!tourney/Prizes#tournament-screening) for more information about tournament screening.
+
+5. All players eligible to participate in the tournament will be published after the Registration period ends.
+
+### Stage Instructions
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:540]:
```suggestion
### Stage instructions
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 30]
```diff
@@ -0,0 +1,656 @@
...
+
+    - The Group stages will be best of 7 (first team to 4 wins).
+        
+    - The Quarterfinals and Semifinals will be best of 9 (first team to 5 wins).
+        
+    - The Finals will be best of 11 (first team to 6 wins).
+        
+    - The Grand Finals will be best of 13 (first team to 7 wins).
+
+### Match Instructions
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:580]:
```suggestion
### Match instructions
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 31]
```diff
@@ -0,0 +1,656 @@
...
+
+5. Beatmap selection will alternate between each player/team selecting a beatmap out of the mappool.
+
+6. Players map pick freely from any mod pool.
+
+   - In the case of a tie, the tiebreaker beatmap will be played.
+
+7. The results will be published via a statistics sheet.
+
+### Mappool Instructions
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:610]:
```suggestion
### Mappool instructions
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 32]
```diff
@@ -0,0 +1,656 @@
...
+
+8. The FreeMod bracket will have Free Mods enabled.
+
+   - Possible mod choices are Hidden, HardRock and Hidden+HardRock.
+
+9. The tiebreaker will be played under FreeMod conditions.
+
+   - When playing the tiebreaker, there are no mod requirements.
+
+### Scheduling Instructions
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:638]:
```suggestion
### Scheduling instructions
```
Sentence case

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 33]
```diff
@@ -0,0 +1,656 @@
...
+
+3. Players may ask for a reschedule upon agreement with the opponent(s) if the current schedule does not work for them.
+
+4. Reschedules will only be considered if both players/teams agree to a time at least 24 hours before the first match in that particular stage.
+
+   - If players wish to change the scheduled time within 24 hours until the match, they have to seek for an available referee for the rescheduled time. **Participants cannot use this opportunity more than once.**
+
+5. If it is certain that participant(s) is/are unavailable for the scheduled time, they are obliged to inform the tournament staff beforehand. This is required in order not to waste the time of staff and other participants.  
+
+[flag_TR]: /wiki/shared/flag/TR.gif
```
Comment (fajar13k) [inline wiki/Tournaments/112/Summer_2019/en.md:652]:
```suggestion
[flag_TR]: /wiki/shared/flag/TR.gif "Turkey"
```
Caption the flag, and make sure to Ascend Sort it. (A -Z) by the flag
e.g.
flag_ID
flag_JP
flag_KR
etc.

File: wiki/Tournaments/en.md [hunk 34]
```diff
@@ -149,6 +149,16 @@ Unofficial tournaments/competitions hosted by the communities.
 | [nik's Winter Tour 2019](NT/NWT_2019) | 2019-01-19 | 2019-03-10 | ![][flag_US] [Fish-](https://osu.ppy.sh/users/3318654) | ![][flag_AU] [loveleft](https://osu.ppy.sh/users/9240047) | ![][flag_DE] [\[Lucky\]](https://osu.ppy.sh/users/1303685) |
 | [nik's Spring/Summer Tour 2019](NT/NSST_2019) | 2019-04-27 | 2019-06-29 | ![][flag_GB] [Bubbleman](https://osu.ppy.sh/users/5182050) (Tier 1), ![][flag_KR] [Civil oath](https://osu.ppy.sh/users/3216107) (Tier 2) | ![][flag_US] [fieryrage](https://osu.ppy.sh/users/3533958) (Tier 1), ![][flag_KR] [\_Suyong](https://osu.ppy.sh/users/7295276) (Tier 2) | ![][flag_US] [im a fancy lad](https://osu.ppy.sh/users/4908650) (Tier 2), ![][flag_NL] [Swerro](https://osu.ppy.sh/users/4507667) (Tier 2) |
 
+### ![](/wiki/shared/mode/osu.png) 112's Tournaments
```
Comment (fajar13k) [inline wiki/Tournaments/en.md:152]: Also put it under Yuri Cup since it's the latest addition to the index.

**Comment by zeusminus** [comment]
> Lowercase all file names.

@fajar13k is there a way to change the file names, or do I have to recreate them?

**Comment by fajar13k** [comment]
You can open your IDE (Visual Studio Code for example and rename it there) ? ?  ?

File: wiki/Tournaments/112/Summer_2019/en.md [hunk 35]
```diff
@@ -0,0 +1,580 @@
+---
+tags:
+- 112servis
+- 112ST
+- 112ST 2019
+---
+
+# 112's Summer Tournament 2019
+
+![112ST Logo](img/logo.png)
```
Comment (MegaApplePi) [inline wiki/Tournaments/112/Summer_2019/en.md:10]: it looks like this image has these lines in the transparent area

image

was this intended?

Comment (zeusminus) [reply wiki/Tournaments/112/Summer_2019/en.md:10]: I don't think so, will clear those lines and replace the image

**Pull request merged**